PlayCover/de.lproj/Localizable.strings

```diff
@@ -1,44 +1,93 @@
-"This .IPA is corrupted. It doesn't contain Info.plist." = "Diese .IPA ist beschädigt. Sie enthält keine Info.plist.";
-"It is impossible to decrypt .IPA with SIP disabled. Please, enable it." = "Es ist unmöglich, .IPA mit deaktiviertem SIP zu entschlüsseln. Bitte aktiviere es.";
-"Please reinstall PlayCoverApp" = "Bitte installiere die PlayCoverApp neu";
-"Make sure you don't disallowed PlayCover to access files in Settings - Secuirity & Privacy" = "Stelle sicher, dass Du PlayCover den Zugriff auf Dateien in den Einstellungen – Sicherheit und Datenschutz nicht untersagt hast";
-"This .IPA is courrupted.Try to use another .ipa" = "Diese .IPA ist beschädigt.Versuch, eine andere .ipa . zu verwenden";
-"You have not performed all instructions! Please, perform all instructions in 'Settings' or watch video in pinned messages." = "Du hast nicht alle Anweisungen ausgeführt! Bitte führe alle Anweisungen unter 'Einstellungen' aus oder schau Dir das Video in den angepinnten Nachrichten an..";
-"Get" = "Herunterladen";
-"soundAlert.messageText" = "";
-"soundAlert.informativeText" = "";
-"soundAlert.successText" = "";
+"app.add" = "App hinzufügen";//
+"search.search" = "Search...";
+"app.settings" = "Einstellungen";//
+"app.showInFinder" = "Im Finder anzeigen";//
+"app.openCache" = "Open app data";
+"app.clearCache" = "App-Cache leeren";//
+"app.importkm" = "Import keymapping";
+"app.exportkm" = "Export keymapping";
+"app.delete" = "App löschen";//
+"alert.appCacheCleared" = "App data has been cleared!";
+"alert.kmImported" = "Keymapping imported!";
+"alert.errorImportKm" = "Error occured when importing!";
+"alert.app.delete" = "All app data will be erased. You may need to download app files again. Do you wish to continue?";
+"alert.wrongFileType" = "Falscher Dateityp";//
+"button.OK" = "OK";
+"button.Proceed" = "Proceed";
+"button.Cancel" = "Cancel";
+"button.Close" = "Close";
+"button.Dismiss" = "Dismiss";
+"error.corruptedIPA" = "Diese .IPA ist beschädigt. Sie enthält keine Info.plist.";//
+"error.waitInstallation" = "Please wait for the current installation to finish!";
+"error.reinstallApp" = "Bitte installiere die PlayCover neu";
+"menubar.downloadMoreApps" = "Download more apps";
+"menubar.downloadMoreApps.help" = "Use this site to download apps for installation";
+"menubar.exportToSideloady" = "Export to Sideloadly";
+"menubar.exportToSideloady.help" = "Export IPAs for Sideloady from sideloady.io which does not need SIP to be disabled for apps and games to work.";
+"alert.wrongFileType.message" = "Choose an .ipa file";
+"app.add.help" = "Drag or open an app file to install. IPAs from Configurator or iMazing won't work! You should get decrypted IPAs, either from the top right button, Discord, AppDb, or a jailbroken device.";
+"app.update" = "Update app";
+"bottomBar.notices" = "Notices";
+"alert.followInstructionAndRestartApp" = "Please follow the given instructions, and restart the App.";
```

Comment:
Please consolidate all `alert.*` locals in one place. They're too scattered around rn.